canonical-websites/snapcraft.io
# Add ability to dev to update URL from env var.

# Summary

Now developpers can use other urls (for example stagings urls) by adding them in the .env file.

# QA

- `./run`
- http://0.0.0.0:8004/search?q=no
- http://0.0.0.0:8004/account
- http://0.0.0.0:8004/logout
- http://0.0.0.0:8004/discover
- All those pages should work as usual
- Now stop the run script
- In the .`.env` file add those lines:
```
SNAPCRAFT_IO_API=https://api.staging.snapcraft.io/api/v1/
DASHBOARD_API=https://dashboard.staging.snapcraft.io/dev/api/
SEARCH_API=https://search.apps.staging.ubuntu.com/api/v1/
LOGIN_URL=https://login.staging.ubuntu.com
```
- `./run`
- http://0.0.0.0:8004/search?q=no
- http://0.0.0.0:8004/account
- http://0.0.0.0:8004/logout
- http://0.0.0.0:8004/discover
- You should have access to all those pages, but the data displayed should be the data from staging.
